### PR TITLE
Docs/2.3

### PR DESCRIPTION
**Pull request by GodloveD**
Changes proposed in this pull request

 - added a whole slew of new asciicasts
 - flipped order of create and bootstrap in side-bar to follow logical progression
 - edited the wording, tried to be more terse
 - removed a bunch of `bash` labels that were adding inaprop syntax highlighting to terminal commands
 - added some new content 

@singularityware-admin

### CODE REVIEW

**Comment by vsoch** [comment]
ooh lala! Previewing now...

File: _data/sidebars/user_docs.yml [hunk 1]
```diff
@@ -19,14 +19,14 @@ entries:
...
-    - title: Bootstrap an Image
-      url: /bootstrap-image
-      output: web, pdf
-
     - title: Create an Image
       url: /create-image
       output: web, pdf
 
+    - title: Bootstrap an Image
+      url: /bootstrap-image
```
Comment (vsoch) [inline _data/sidebars/user_docs.yml:27]: yes I like this change a lot, users should create before bootstrap!

**Comment by vsoch** [comment]
oh my gosh, it's beautiful!

**Comment by GodloveD** [comment]
Thanks!  I'm glad you like it!  I hope to do more.

**Pull request merged**